src/app/app.component.ts

```diff
@@ -35,8 +34,7 @@ export class AppComponent implements OnInit, OnDestroy {
           (status: InteractionStatus) => status === InteractionStatus.None,
         ),
         first(),
-        switchMap(() => this.userService.initUser()),
-        first(),
+        tap(() => this.store.dispatch(UserActions.initUser())),
```

Comment:
Aktuell kommt man auf das UI auch wenn die Registrierung via initUser noch nicht abgeschlossen oder erfolglos ist. Das sollte nicht möglich sein. Solange keine Response vom BE zurückkommt, sollte meiner Meinung nach das UI komplett ausgeblendet sein und nur der Loader dargestellt werden. Im Fall eines Fehlers sollte nur eine Error-Meldung angezeigt werden, inkl. Status aus der Response, falls vorhanden.

Reply:
Ich habe auf jetzt einen Loader eingefügt für die gesamte App bevor der User angekommen ist mit HTTP Error anzeigen bei einem fehlschlagen der Post request.

Bei einem neu einloggen gibt es leider bisschen hin und her in der UI die ich mir nicht ganz erklären kann. Da bräuchte ich deine Hilfe!

Vielen Dank dir :)